windows/installer/MozillaVPN.wxs

```diff
@@ -91,7 +91,9 @@
 					Start="auto"
 					Arguments="windowsdaemon"
 					ErrorControl="normal"
-					Vital="yes" />
+					Vital="yes">
+					 <util:PermissionEx User="Users" ServiceQueryStatus="yes"  ServiceStart="yes" />
```

Comment:
Does it mean that each app can stop/start the service? This is what I don't like too much...
is there anything better we can do?

Reply:
This only means, checkstate && start but other user apps cant stop it :)